lib/filewatch/discoverer.rb

```diff
@@ -56,13 +57,13 @@ def discover_files_ongoing(path)
     end
 
     def discover_any_files(path, ongoing)
-      fileset = Dir.glob(path).select{|f| File.file?(f)}
-      logger.trace("discover_files",  "count" => fileset.size)
+      fileset = Dir.glob(path).select { |f| File.file?(f) }
+      logger.trace("discover_files", :count => fileset.size)
       fileset.each do |file|
-        pathname = Pathname.new(file)
         new_discovery = false
-        watched_file = @watched_files_collection.watched_file_by_path(file)
+        watched_file = @watched_files_collection.get(file)
```

Comment:
:+1: but for consistency I am going to keep it as is, esp. since bellow (in the same method)
we already have the `@` i-var style access: 
```ruby
            # now add the discovered file to the watched_files collection and adjust the sincedb collections
            @watched_files_collection.add(watched_file)
```
also, potentially the `attr_reader` could be removed - not sure it's necessary for the outside.